monitoring/collect_and_push_job_stats.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import argparse
+import getpass
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from stats import collect_and_push_job_stats
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--job-name', required=True)
+    parser.add_argument('--build-url', required=True)
+    parser.add_argument('--mysql-host', required=True)
+    parser.add_argument('--mysql-username', required=True)
+    parser.add_argument('--mysql-password', required=False)
+    parser.add_argument('--mysql-database', required=True)
+    args = parser.parse_args()
+
+    if not args.mysql_password:
+        prompt = 'Enter password (for MySQL user {}): '.format(args.mysql_username)
+        args.password = getpass.getpass(prompt=prompt)
+
+    return args
+
+
+def get_mysql_connection_string(host, username, password, database):
```

Comment:
Please move this function to some `common` file. Another good idea is to make it a method of `MysqlCommonArgumentParser` class.